def test_command_line_handling_do_discovery_too_many_arguments(self):


    class Stop(Exception):
        pass

    def usageExit():
        raise Stop
    program = object.__new__(unittest2.TestProgram)
    program.usageExit = usageExit
    self.assertRaises(Stop, (lambda : program._do_discovery(['one', 'two', 'three', 'four'])))
